travel-2018.html

```diff
@@ -0,0 +1,235 @@
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-111617773-1"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag('js', new Date());
+    gtag('config', 'UA-111617773-1');
+  </script>
+
+  <style type="text/css">
+    li {
+      list-style: initial;
+  }
+  </style>
+
+
+  <!-- Fonts -->
+  <link href="custom/css/animate.css" rel="stylesheet" />
+  <link href="custom/css/style.css" rel="stylesheet">
+  <link href="custom/css/scroll.css" rel="stylesheet">
+  <link href="custom/css/buttons.css" rel="stylesheet">
+  <link rel="shortcut icon" href="img/favicon.png">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat+Alternates:300,400,500,600" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+</head>
+
+
+<body id="page-top" data-spy="scroll" data-target=".navbar-custom">
+  <!-- Preloader -->
+  <!-- <div id="preloader">
+    <div id="load"></div>
+  </div> -->
+
+  <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+
+      <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
+        <img src="./img/banner-logo.png" class="logo" alt="banner-logo" style="width: 150px; display: inline-block;">
+      </a>
+
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"
+            aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav nav-justified">
+          <li><a href="index.html#home">Home</a></li>
+          <li><a href="index.html#journey">Journey</a></li>
+          <li><a href="index.html#speakers">Speakers</a></li>
+          <!-- <li><a href="#conference">Schedule</a></li> -->
+          <li><a href="index.html#sponsors">Sponsors</a></li>
+          <li><a href="index.html#tickets">Tickets</a></li>
+          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
+          <li><a href="index.html#venue">Venue</a></li>
+          <li><a href="faq.html">FAQ</a></li>
+          <li><a href="volunteer.html">Volunteer</a></li>
+          <li class="active"><a href="travel-2018.html">Travel</a></li>
+          <li><a href="coc.html">COC</a></li>
+        </ul>
+      </div>
+
+  </nav>
+
+
+  <!-- Section: about -->
+
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
```

Comment:
It would be better with `PyCon India`. What do you guys think? @ananyo2012 @vipulgupta2048

Reply:
Yes, its more specific

Reply:
cc: @aswanipranjal

Reply:
@aswanipranjal Make it. 
`How to get to PyCon India 2018?` or `Getting to PyCon India 2018`

These would be more appropriate. Choose whichever you prefer.